app/src/app/utils/GeometryWorker/geometryWorker.types.ts

```diff
@@ -70,9 +70,10 @@ export type GeometryWorkerClass = {
    * @returns The centroids and dissolved outlines of the parsed features within the view.
    */
   getCentroidsFromView: (props: {
-    bounds: [number, number, number, number];
-    activeZones: number[];
-    fast?: boolean;
+    bounds: [number, number, number, number],
+    activeZones: number[],
+    fast?: boolean,
```

Comment:
PP: Prefer to avoid boolean flags (though of course I do them too!). Would prefer descriptive strategy pattern e.g.

```suggestion
    labelPlacementMethod?: LabelPlacementMethod,
```

where `LabelPlacementMethod` is an enum e.g.

```ts
enum LabelPlacementMethod {
    NonCollidingCentroids = (...props) => void, // whatever this should be
    CenterOfMass = (...props) => void, // whatever this should be
    // etc.
}
```

Sidebar: I've been thinking we should start to implement some feature flags so we can test these different ideas in prod w/o blocking PRs while allowing the features to bake for a bit.

Reply:
here's an issue for the feature flag idea #308

Reply:
Agreed on feature flags and avoiding boolean flags!

In this case passing a function directly to the `GeometryWorker` won't quite work since it gets serialized, but using specified strings and internal methods to the worker (eg. `getCentroidsFromView(bounds: number[], strategy: 'centerOfMass' | 'nonCollidingCentroids') should have the same effect!